### PR TITLE
fix: speakers links and bio

### PR DESCRIPTION
**Pull request by dotdc**
Noticed the speaker link was broken for "Yet Another Streaming PromQL Engine" (recently replaced in https://github.com/promcon/website/pull/290).
Also took this opportunity to add my twitter and remove a duplicate entry in my bio.

### CODE REVIEW

**Comment by dotdc** [comment]
cc @metalmatze

File: content/2023-berlin/speakers/david-calvert.md [hunk 1]
```diff
@@ -2,8 +2,6 @@
 title: "David Calvert"
 ---
 
-## David Calvert
-
-Working at Oqton
```
Comment (metalmatze) [inline content/2023-berlin/speakers/david-calvert.md:7]: We always have that as a first line of the biography. Let's be consistent even though it's mentioned in your text too.

File: content/2023-berlin/talks/yet-another-streaming-promql-engine.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: "Yet Another Streaming PromQL Engine"
 
 ## Yet Another Streaming PromQL Engine
 
-Speaker(s): [Charles Korn](../../speakers/charles-korn)
+Speaker(s): [György Krajcsovits](../../speakers/gyorgy-krajcsovits)
```
Comment (metalmatze) [inline content/2023-berlin/talks/yet-another-streaming-promql-engine.md:7]: Oops. Thanks!, 👍

File: content/2023-berlin/speakers/david-calvert.md (hunk 1 above)
Comment (dotdc) [reply content/2023-berlin/speakers/david-calvert.md:7]: Sorry, I didn't notice the pattern, just swapped :+1:

**Pull request merged**